src/Payments/ApplePay.res

```diff
@@ -54,7 +54,7 @@ let make = (
 
   let isGuestCustomer = UtilityHooks.useIsGuestCustomer()
 
-  let processPayment = bodyArr => {
+  let processPayment = (bodyArr, isThirdPartyFlow) => {
```

Comment:
Could you please keep the default value as false, and only pass as true where it is a thirdPartyFlow